cli/plugins/audit_plugin/app.py

```
@@ -13,53 +13,91 @@
+        self, records: list[dict[str, Any]], positions: str | None
+    ) -> tuple[dict[str, Any], dict[str, Any]]:
+        """Select the two audit trails to compare from the loaded records."""
+        if positions:
+            try:
+                parsed_positions = _parse_positions(positions, len(records))
+            except ValueError:
+                console.print(
+                    "[red]Invalid positions. Please specify two numbers "
+                    f"between 1 and {len(records)}[/red]"
+                )
+                raise typer.Exit(1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)